### ISSUE TITLE
Build error after updating to latest nightly

### ISSUE THREAD

**Issue by alexschneider** [hyperium/hyper#341]
```
 Downloading cookie v0.1.13
 Downloading time v0.1.19
 Downloading log v0.2.5
 Downloading gcc v0.3.1
   Compiling gcc v0.3.1
   Compiling log v0.2.5
   Compiling openssl-sys v0.4.3
   Compiling url v0.2.21
   Compiling mime v0.0.8
   Compiling time v0.1.19
   Compiling openssl v0.4.3
   Compiling cookie v0.1.13
   Compiling hyper v0.2.0
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:36:10: 36:18 error: missing documentation for an associated type
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:36     type Acceptor: NetworkAcceptor;
                                                                                              ^~~~~~~~
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                            ^~~~~~~~~~~~
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:43:10: 43:16 error: missing documentation for an associated type
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:43     type Stream: NetworkStream + Send + Clone;
                                                                                              ^~~~~~
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                            ^~~~~~~~~~~~
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:92:10: 92:16 error: missing documentation for an associated type
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:92     type Stream: NetworkStream + Send;
                                                                                              ^~~~~~
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                            ^~~~~~~~~~~~
error: aborting due to 3 previous errors
Could not compile `hyper`.
```

I ran `cargo clean` and `cargo update` and it still didn't work properly

**Comment by mindcat** [comment]
```
╭─mindcat@mindcat-linux-pc ~/workspace/nekoneko  ‹master*› 
╰─➤  cargo build -j 4
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling hyper v0.2.0
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:36:10: 36:18 error: missing documentation for an associated type
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:36     type Acceptor: NetworkAcceptor;
                                                                                                 ^~~~~~~~
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                               ^~~~~~~~~~~~
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:43:10: 43:16 error: missing documentation for an associated type
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:43     type Stream: NetworkStream + Send + Clone;
                                                                                                 ^~~~~~
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                               ^~~~~~~~~~~~
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:92:10: 92:16 error: missing documentation for an associated type
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/net.rs:92     type Stream: NetworkStream + Send;
                                                                                                 ^~~~~~
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3:9: 3:21 note: lint level defined here
/home/mindcat/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.2.0/src/lib.rs:3 #![deny(missing_docs)]
                                                                                               ^~~~~~~~~~~~
error: aborting due to 3 previous errors
Could not compile `hyper`.

To learn more, run the command again with --verbose.
╭─mindcat@mindcat-linux-pc ~/workspace/nekoneko  ‹master*› 
╰─➤  rustc -V                                                                                                                                 101 ↵
rustc 1.0.0-nightly (4db0b3246 2015-02-25) (built 2015-02-26)
```

**Comment by seanmonstar** [comment]
Fixed via #340
